### PR TITLE
Column with cards

### LINKED ISSUE #1: Column with cards
image

### PR DESCRIPTION
**Pull request by ShatterPlayer**
Resolves #1.
image

### CODE REVIEW

**Comment by Spookyless** [comment]
`Column` component requires `name` prop, but you didn't update the place where they're used (`app/page.jsx`)

File: components/Column/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  {
+    name: 'Zadanie 3',
+    date: '2021-10-01',
+  },
+  {
+    name: 'Zadanie kolejne'
+  }
+]
+
+interface ColumnProps {
```
Comment (rubikon02) [inline components/Column/index.tsx:54]: Column should have:
- type - enum ('todo', 'in progress', 'done') or null for milestone tracking
- optional description (text)
- order (int)

File: components/Task/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import { Avatar, Button, Chip, Paper, Tooltip, Typography } from '@mui/material'
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+
+interface Label {
+  name: string
+  color: string
+}
+
+interface TaskProps {
```
Comment (rubikon02) [inline components/Task/index.tsx]: Task should have order (int)

File: components/Task/index.tsx
Comment (rubikon02) [inline components/Task/index.tsx:1]: - Rename to card
- Change naming to differentiate between column card and modal card

**Review by rubikon02 — CHANGES_REQUESTED** [review]

File: components/Column/index.tsx (hunk 1 above)
Comment (Spookyless) [reply components/Column/index.tsx:54]: Order should not be a prop - the column doesn't need to know what order it is, displaying them in correct order is a job of the project view

File: components/Task/index.tsx (hunk 2 above)
Comment (Spookyless) [reply components/Task/index.tsx]: No order, again

File: components/Task/index.tsx
Comment (ShatterPlayer) [reply components/Task/index.tsx:1]: Task component has been renamed to ColumnCard (134a86aab4e6bd8af361d8a2886c1b1c3dda3a62)

**Review by Spookyless — APPROVED** [review]

**Review by rubikon02 — APPROVED** [review]

**Pull request merged**